modules/extra/images.yml

```diff
@@ -244,6 +244,15 @@ images:
       - "v0.3.0"
       - "v0.4.0"
       - "v0.5.0"
+      
+    destinations:
+      - registry.sighup.io/fury/jimmidyson/configmap-reload
+
+  - name: Configmap reloader
+    source: docker.io/jimmidyson/configmap-reload
+    tag:
+      - "v0.13.1"
+
```

Comment:
please review this change, I don't think it is correct. You seam to be duplucating the entry instead of adding a new tag

Reply:
It was a typo that duplicated the image object; it is the same destination but from a different source. I’ve updated the source.